packages/focusmanager/src/react/use-focus-manager.ts

```diff
@@ -13,35 +14,43 @@ const FOCUSABLE_SELECTORS = [
   'video[controls]'
 ]
 
-const defaultOptions = {
-  autofocus: true,
-  returnFocus: true,
-  trapped: true
-}
-
-export default function useFocusManager(ref, _options) {
-  const opts = { ...defaultOptions, ..._options }
-
-  const prevActiveElRef = React.useRef()
+export default function useFocusManager(
+  ref: React.RefObject<HTMLDivElement>,
+  {
+    autofocus,
+    returnFocus,
+    trapped
+  }: {
+    autofocus: boolean
+    returnFocus: boolean
+    trapped: boolean
+  }
+) {
+  const prevActiveElRef = React.useRef<HTMLElement>(null)
   React.useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
just merged an eslintrc update that allows ts-comments, but it requires a commented reason.  Eg,

```
@ts-ignore: this is the reason why
```